pkg/bb/bb.go

```diff
@@ -289,23 +293,26 @@ func SrcFiles(p *build.Package) []string {
 // RewritePackage rewrites pkgPath to be bb-mode compatible, where destDir is
 // the file system destination of the written files and bbImportPath is the Go
 // import path of the bb package to register with.
-func RewritePackage(env golang.Environ, pkgPath, bbImportPath string, importer types.Importer) error {
+func RewritePackage(env golang.Environ, pkgPath, bbImportPath string, importer types.Importer) (bool, error) {
```

Comment:
Question: how this extra return fixes it for plan9 ? and what was the issue ?

addition of this bool return seems the main fix here, but seems it is false whenever an error is returned, but forcing a true regardless of return of final p.Write

Reply:
It is the commit messsage

Reply:
does this make sense now or is it still unclear? Thanks!

Reply:
> the basic idea of the pkg/bb change: a parse error is an error, and should cause the u-root commadn to fail. ok means that it found code for this GOOS GOARCH and it had a main. It's not problem to not have a main for a given GOOS GOARCh any more; this happens when, e.g., we try to build for a non-supported OS

makes sense to me